packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -4,23 +4,82 @@ import { error_types, unrecoverable_errors, observer, message_types } from '@der
 import { contract_stages } from '../constants/contract-stage';
 import { switch_account_notification } from '../utils/bot-notifications';
 
+const statistics_storage_key = 'statistics_cache';
+
+const empty_statistic = {
+    lost_contracts: 0,
+    number_of_runs: 0,
+    total_profit: 0,
+    total_payout: 0,
+    total_stake: 0,
+    won_contracts: 0,
+};
+let get_statistics;
```

Comment:
- All these variables can live inside the class constructor, there's no need to have them outside.
- `empty_statistics` suggests we're dealing with a single statistic here, which we don't, please use `empty_statistics`, in addition, since the values never change I suggest wrapping the object in `Object.freeze` so we don't accidentally modify it.
- Alternatively, create a function that returns a new object each time.